ci_scripts/build_macos.sh

```diff
@@ -5,6 +5,13 @@
 # exit if any command fails
 set -e
 
+# Travis CI doesn't java_home for some reason
+if [ -z "$JAVA_HOME" ]; then
+  echo Setting JAVA_HOME from /usr/libexec/java_home
+  export JAVA_HOME=$(/usr/libexec/java_home)
+  echo JAVA_HOME is $JAVA_HOME
+fi
+
```

Comment:
Yeah, so there was a failure to find libktx_jni.

I changed the CI to add `/Debug` to the build folder to find libktx_jni. It should find the library now. That directory isn't listed in `java.library.path` since it isn't a standard installation directory.